sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/implementation/models/KeyRotationPolicyAttributes.java

```diff
@@ -22,43 +22,28 @@
 
     /**
      * Get the optional key expiration period used to define the duration after which a newly rotated key will expire.
-     * It should be defined as an ISO 8601 duration. For example, 90 days would be formatted as follows: "P90D", 3
-     * months would be "P3M", 48 hours would be "PT48H" and 1 year and 10 days would be "P1Y10D".
+     * It should be at least 28 days and should be defined as an ISO 8601 duration. For example, 90 days would be
+     * formatted as follows: "P90D", 3 months would be "P3M" and 1 year and 10 days would be "P1Y10D".
```

Comment:
Nit:
```suggestion
     * It should be at least 28 days and should be defined as an ISO 8601 duration. For example, 90 days would be
     * "P90D", 3 months would be "P3M" and 1 year and 10 days would be "P1Y10D".
```
I also think it would be a good idea to link to the Wikipedia documentation of ISO 8601 durations (https://wikipedia.org/wiki/ISO_8601#Durations) somewhere -- I think JS just does in a sample, but Python includes the link in docstrings as well

Reply:
Thanks McCoy, that can definitely help understand how to build these strings.